GChan/Forms/SettingsForm.cs

```diff
@@ -148,14 +149,14 @@ private bool hasWriteAccessToFolder(string folderPath, out string reason)
 
         private void SetPathButton_Click(object sender, EventArgs e)
         {
-            FolderBrowserDialog FolD = new FolderBrowserDialog();
-            FolD.Description = "Select Folder";
-            FolD.SelectedPath = @"C:\";
-            DialogResult dRes = FolD.ShowDialog(this);
-
-            if (dRes == DialogResult.OK)
+            var openFileDialog = new CommonOpenFileDialog();
+            openFileDialog.IsFolderPicker = true;
+            openFileDialog.Title = "Select Folder";
+            openFileDialog.InitialDirectory = @"C:\";
```

Comment:
Good opportunity to replace this with `Path.GetPathRoot(Environment.SystemDirectory);`.
Not every computer has a C drive I have learned. 
Or maybe this `Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments)`